src/services/user/services/AdminUsers.js

```diff
@@ -275,25 +275,26 @@ class AdminUsers {
 	}
 
 	async remove(id, params) {
-		const { _ids } = params.query;
-		const currentUser = await getCurrentUserInfo(params.account.userId);
-
+		const _ids = [];
 		if (id) {
-			const userToRemove = await getCurrentUserInfo(id);
-			if (!equalIds(currentUser.schoolId, userToRemove.schoolId)) {
-				throw new Forbidden('You cannot remove users from other schools.');
+			if (typeof id === 'object') {
+				_ids.push(id._id);
+			} else {
+				_ids.push(id);
 			}
-			await this.app.service('accountModel').remove(null, { query: { userId: id } });
-			return this.app.service('usersModel').remove(id);
+		} else {
+			_ids.push(...(((params || {}).query || {})._ids || []));
 		}
 
+		const currentUser = await getCurrentUserInfo(params.account.userId);
 		const usersIds = await Promise.all(_ids.map((userId) => getCurrentUserInfo(userId)));
-		if (usersIds.some((user) => !equalIds(currentUser.schoolId, user.schoolId))) {
+		if (!usersIds.some((user) => equalIds(currentUser.schoolId, user.schoolId))) {
 			throw new Forbidden('You cannot remove users from other schools.');
 		}
 
-		await this.app.service('accountModel').remove(null, { query: { userId: { $in: _ids } } });
-		return this.app.service('usersModel').remove(null, { query: { _id: { $in: _ids } } });
+		return Promise.all(
+			_ids.map((userId) => this.app.service(`users/v2/admin/${this.roleName}`).remove(userId, params))
```

Comment:
only for debug, 
const removePromises = _ids.map()
Promise.all(removePromises); is helpful